<launch>
    <!-- <param name="use_sim_time" value="true"/> -->
    <param name="config_path" value="/home/lwh/code/graph_3d_slam/src/LifelongBackend/config/param.yaml"/>
    <param name="database_path" value="/home/lwh/SlamDataBase/"/>
    
    <!-- <node name="nmea_topic_driver" pkg="nmea_navsat_driver" type="nmea_topic_driver" output="screen" /> -->    
    <node name="backend_node" pkg="lifelong_backend" type="backend_node" output="screen" >
    </node>
</launch>